Content.Server/Administration/Systems/AdminSystem.cs

```diff
@@ -21,7 +21,7 @@ public sealed class AdminSystem : EntitySystem
         [Dependency] private readonly IPlayerManager _playerManager = default!;
         [Dependency] private readonly IAdminManager _adminManager = default!;
 
-        private readonly Dictionary<NetUserId, PlayerInfo> _playerList = new();
```

Comment:
I think keeping it private is better and adding a method to get the relevant info.